consensus/polybft/polybft.go

```diff
@@ -118,6 +119,9 @@ type Polybft struct {
 
 	// tx pool as interface
 	txPool txPoolInterface
+
+	// msgHandlers contains IBFT consensus messages handlers
+	msgHandlers []IBFTMessageHandler
```

Comment:
Maybe rename to `ibftMsgHandlers`?

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1756/commits/9699327c17a940f08b201d0d0408cab6879d50f4